### PR TITLE
Add question 8: offer community support

### PR DESCRIPTION
**Pull request by theseanything**
https://trello.com/c/wGp7MIda/29-implement-question-8

### CODE REVIEW

File: app/controllers/coronavirus_form/offer_community_support_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      radio: offer_community_support,
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      render "coronavirus_form/#{PAGE}"
+    elsif session["check_answers_seen"]
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    else
+      redirect_to controller: "coronavirus_form/thank_you", action: "show"
```
Comment (bilbof) [inline app/controllers/coronavirus_form/offer_community_support_controller.rb]: This'll need to become the route to 'can you offer other kinds of support'

File: app/controllers/coronavirus_form/offer_community_support_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    end
+  end
+
+private
+
+  PAGE = "offer_community_support"
+
+  def previous_path
+    coronavirus_form_expert_advice_path
```
Comment (bilbof) [inline app/controllers/coronavirus_form/offer_community_support_controller.rb]: This should be the path to 'can you offer social care'?

Comment (theseanything) [reply app/controllers/coronavirus_form/offer_community_support_controller.rb]: Yup! waiting for the question 7 PR to be merged

File: app/controllers/coronavirus_form/offer_community_support_controller.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    else
+      redirect_to controller: "coronavirus_form/#{NEXT_PAGE}", action: "show"
+    end
+  end
+
+private
+
+  PAGE = "offer_community_support"
+  NEXT_PAGE = "thank_you"
```
Comment (bilbof) [inline app/controllers/coronavirus_form/offer_community_support_controller.rb:33]: Could we change this to 'offer any other kind of support' (and add a route)? If you're doing this in q9 already no worries about this 👍

File: config/locales/en.yml [hunk 4]
```diff
@@ -184,6 +184,15 @@ en:
...
       custom_select_error: Select whether you can offer care
+    offer_community_support:
+       title: Can you offer community support?
+       hint: For example distributing food or helping homeless people.
+       options:
+         option_yes:
+           label: "Yes"
+         option_no:
+           label: "No"
+       custom_select_error: Select which services you can provide
```
Comment (bilbof) [inline config/locales/en.yml:195]: You might want a more specific error message here, but no worries if you're going to update the content later

File: spec/controllers/coronavirus-form/offer_community_support_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  describe "GET show" do
+    it "renders the form" do
+      get :show
+      expect(response).to render_template(current_template)
+    end
+  end
+
+  describe "POST submit" do
+    let(:selected_yes) { I18n.t("coronavirus_form.offer_care.options.option_yes.label") }
+    let(:selected_no) { I18n.t("coronavirus_form.offer_care.options.option_no.label") }
```
Comment (bilbof) [inline spec/controllers/coronavirus-form/offer_community_support_controller_spec.rb]: It looks like `selected_no` isn't used here

**Review by bilbof — APPROVED** [review]
Looks great :shipit:

File: spec/controllers/coronavirus-form/offer_community_support_controller_spec.rb (hunk 5 above)
Comment (huwd) [reply spec/controllers/coronavirus-form/offer_community_support_controller_spec.rb]: this is true! I could remove it, will immediately add a test for both yes and no, when i merge q9

Comment (huwd) [reply spec/controllers/coronavirus-form/offer_community_support_controller_spec.rb]: and that test will use :selected_no

**Pull request merged**